File: week-02/day-01/draw_pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was
c=5
k = "*"
j="h"
d=5//2

for i in range(0,d):
    t=c-(2*d)
    print(j*d+t*k+j*d)
    
    

File: week-08/day-01/29_draw_pyramid/draw_pyramid.js
var lineCount = 4;
var char = "*";
// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

//draw_dymond
var lineCount = 4;
var stars = "";
var spaces = "";
var j = 1;
for(i=0; i<= lineCount; i++){
    stars += "*".repeat(j);
    spaces += " ".repeat(lineCount-i);
    console.log(spaces + stars);
    stars = ""
    spaces = ""
    j += 2;
}